library/src/main/java/com/cube/storm/ui/quiz/view/holder/ImageQuizItemViewHolder.java

```diff
@@ -124,14 +126,32 @@ private void selectAnswer(View answerLayout, boolean selectAnswer)
 
 		// Set image label style and its background
 		TextView textView = answerLayout.findViewById(R.id.label);
+		CharSequence labelText = null;
 		if (textView != null)
 		{
 			textView.setTextAppearance(answerLayout.getContext(),
 				selectAnswer ? R.style.QuizImageLabel_Selected : R.style.QuizImageLabel_Unselected);
 			textView.setBackgroundResource(selectAnswer ? R.drawable.quiz_image_label_selected_border : 0);
+			labelText = textView.getText();
 		}
 
-		answerLayout.setContentDescription(textView.getText() + (selectAnswer ? " selected" : " not selected"));
+		// Update the content description based on the selected answer and label text
+		updateAnswerContentDescription(answerLayout, labelText, selectAnswer);
+	}
+	
+	/**
+	 * Updates the view content description changes when an image answer is selected or unselected
+	 * @param answerLayout the layout container other views
+	 * @param baseText the base label text of the view
+	 * @param selectAnswer if answer is selected or unselected
+	 */
+	protected void updateAnswerContentDescription(View answerLayout, @Nullable CharSequence baseText, boolean selectAnswer)
+	{
+		if(baseText == null)
+		{
+			baseText = "";
+		}
+		answerLayout.setContentDescription(baseText + (selectAnswer ? " selected" : " not selected"));
```

Comment:
Would this help work with localisation?

Reply:
Yes, since a user of the library can extend `ImageQuizItemViewHolder` and simply override the `updateAnswerContentDescription` method with their own implementation (which may use localisation)